upi/metal/main.tf

```diff
@@ -184,7 +184,20 @@ resource "aws_route53_record" "ctrlp_int" {
   records = ["${local.ctrp_records}"]
 }
 
-resource "aws_route53_record" "apps" {
+resource "aws_route53_record" "apps_noworker" {
```

Comment:
something we might want to look at, is
- making sure all the nodes, control-plane + compute become part of the *.app DNS
- use route53 health checks to make sure only active nodes with ingress routers become active as DNS response.

Reply:
I'll talk to you on Monday to understand your comments and resolve it. I'm able to successfully bring up a cluster which has both control plane and workers running on the same hosts. Below is the output:
NAME                                                             STATUS   ROLES           AGE   VERSION
master-0.ci-ln-qqgmctk-86010.origin-ci-int-aws.dev.rhcloud.com   Ready    master,worker   16h   v1.17.1
master-1.ci-ln-qqgmctk-86010.origin-ci-int-aws.dev.rhcloud.com   Ready    master,worker   16h   v1.17.1
master-2.ci-ln-qqgmctk-86010.origin-ci-int-aws.dev.rhcloud.com   Ready    master,worker   16h   v1.17.1

Reply:
The suggestion was only a recommendation. this is not important and only for future work

/approve